public/scripts/gif.js

```diff
@@ -0,0 +1,65 @@
+var imagesData = [];
+var imageLinks = [];
+
+function getImageArray(data){
+
+	imagesData = [];
+	$("#gif #gifContent #gifApp").empty();
+	$.each(data, function (index, obj) {
+		if($(obj.event_html).attr("src").indexOf("notcaptured") < 0){
+		imagesData.push(obj);
+		}
+	});
+
+	for(var i = 0; i<imagesData.length; i++)
+	{
+		imageLinks[i] = $(imagesData[i].event_html).attr('src');
+		console.log(imageLinks[i]);
+	}
+
+	var interval = document.getElementById("interval").value;
+
+	createGif(imageLinks, interval);
+
+	document.getElementById("gifButton").addEventListener("click", updateGif);
+}
+
+function updateGif(){
+	$("#gifApp").empty();
+	var interval = document.getElementById("interval").value;
+	createGif(imageLinks, interval);
+}
+	
+
+
+
+function createGif(image, Interval)

```

Comment:
Why upper-case `Interval`?

Reply:
To avoid confusion on 
`interval: Interval,`

Reply:
That is perfectly alright to have them the same. The scope of those identifiers is different and they will not conflict. One should not break the naming convention for things like this. That said, if you really insist to keep them different, you can make the function parameter name more descriptive, such as `animationInterval`.